ui/src/app/package/package-summary/package-summary.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import {PackageSummaryService} from "./package-summary.service";
+import {PackageSummary} from "../../shared/model/package-summary";
+
+@Component({
+  selector: 'app-package-summary',
+  templateUrl: './package-summary.component.html'})
+export class PackageSummaryComponent implements OnInit {
+
+  public packageSummaries: PackageSummary[]
+
+  constructor(private packageSummaryService: PackageSummaryService) { }
+
+  ngOnInit() {
+    this.packageSummaryService.getPackageSummaries().subscribe(data => {
+      this.packageSummaries = data._embedded.packageSummaries;
+    });
```

Comment:
want to be able to handle an error condition so that you can display information back to the users. For example:
https://github.com/spring-cloud/spring-cloud-dataflow-ui/blob/angular/ui/src/app/tasks/task-launch/task-launch.component.ts#L57